detekt.yml

```diff
@@ -132,7 +132,7 @@ complexity:
   LongParameterList:
     active: true
     functionThreshold: 6
-    constructorThreshold: 8
+    constructorThreshold: 9
```

Comment:
Can lower this back to 8 when self match score is removed as we would no longer need `matchService`